framework/contracts/account/proxy/src/commands.rs

```diff
@@ -417,6 +417,9 @@ mod test {
             let msg = ExecuteMsg::IbcAction {
                 msgs: vec![abstract_core::ibc_client::ExecuteMsg::Register {
                     host_chain: "juno".into(),
+                    base_asset: None,
+                    namespace: None,
+                    install_modules: vec![],
                 }],
```

Comment:
Let's add tests for those fields as well! Just one test with all set to make sure it don't panic anywhere unexpectedly